tests/unit/lms/data_tasks/report/create_from_scratch/test_01_functions/test_01_date_functions.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime
```

Comment:
We are stuck between two file naming conventions for the directory name here.

Our standard is to mirror the directory under test, but `01_functions` is not a valid name for a Python module.

Reply:
I'm more familiar with the `test_XX" convention but I reckon we can live with both.